CODING.md

```diff
@@ -0,0 +1,60 @@
+Coding Guidelines
+=================
+
+The Red Hat's FMK project follows [Google Python Style Guide][pyguide] with a
+few additions and exceptions, below.
+
+Code documentation
+------------------
+
+### Wording ###
+Start function descriptions in imperative mood. I.e. "Retrieve Beaker job
+XML", but not e.g. "Retrieves Beaker job XML".
+
+Start class descriptions as if you're labeling canned food. I.e. "Job runner",
+not e.g. "This class runs jobs".
+
+### Formatting ####
+Align argument descriptions on the same column, preferably at tab positions.
+I.e. this:
+
+```python
+        """
+        Initialize a Patchwork REST interface.
+
+        Args:
+            baseurl:        Patchwork base URL.
+            projectname:    Patchwork project name, or None.
+            since:          Last processed patch timestamp in a format
+                            accepted by dateutil.parser.parse. Patches with
+                            this or earlier timestamp will be ignored.
+            apikey:         Patchwork API authentication token.
+        """
+```
+
+Not this:
+
+```python
+        """
```

Comment:
This is inconsistent with both PEP 257 (https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) and Google Python guide which mentions it -- the docstring body should start on the line with triple quotes. We should either follow those, or make the difference clear.

Reply:
Yeah, I forgot to mention that. I really don't like how the "standard" way looks. It's also harder to locate the start of the text with your eyes, when it's stuck to these quotes. Do you think we *can* have that exception?

Reply:
I tried both pydoc and Sphinx and the both seem to work fine:

![pydoc_doc](https://user-images.githubusercontent.com/549246/37952899-00cd19b2-31aa-11e8-9dae-de3d2098243f.png)
![sphinx_doc](https://user-images.githubusercontent.com/549246/37952901-00f87382-31aa-11e8-889e-af807efc80c2.png)

In case we have to use a tool which doesn't it should be possible to reformat the text with a regex replace.

Reply:
LGTM. Can we have this difference documented in the guideline as well (with reasoning about readability), as long as others agree with this change?